### PR TITLE
Remove rest model v1

### LINKED ISSUE #16: Remove model v1
Very soon it will not be needed in Dashboard

### PR DESCRIPTION
**Pull request by Dhaulagiri**
Closes #16

### CODE REVIEW

**Comment by Dhaulagiri** [comment]
@joshwlewis would you be able to give this a once over to see if anything I did jumps out as being obviously problematic?

**Comment by joshwlewis** [comment]
Can we drop the V2 naming? It was so weird to begin with, and there isn't a reason to differentiate anymore.

This should probably include a major version bump too.

Otherwise, LGTM!

**Comment by Dhaulagiri** [comment]
Yeah, let me see if I can do that. And agreed on a major version bump.

File: gulpfile.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var uglifyjs   = require('gulp-uglifyjs');
 var watch      = require('gulp-watch');
 
 gulp.task('default', function() {
-  gulp.src('index.js')
+  gulp.src('index-v2.js')
```
Comment (joshwlewis) [inline gulpfile.js]: i.e. instead of doing this, just `git mv index-v2.js index.js`.

**Pull request merged**